Eunseo week2 task

seulnan opened the pull request:
전체적인 피드백
1. 일단 코드에서 나열심히 공부했어요가 너무 보여서...고생많았어요ㅠㅠㅠ
2. 파일명은 깨질 우려가 있으니 영어로 바꿔봅시다!
3. 폰트나 이미지는 assets폴더로 따로 분리하는 경우가 많아용!
4. 커밋 작은 단위로 한거 너무 잘했어요 커밋 형식만 "[FIX] : 이런거 수정"로 통일해주면 완벽해요!

layout
1. 아주 잘한점: 시맨틱태그 section을 썼다는 점에서 박수 오만개드립니다
2. 주석도 너무 잘 달았어요!!
3. margin 좌우로 auto를 하면 자동으로 중앙정렬되는 점을 이용해보는건 어떨까용?

recipe
1. 백그라운드를 넣는걸 어려워하는것같은데 다른사람코드를 참고해보면서 수정하면 좋아용! 잘짜인 코드를 알맞게 활용하는것도 중요해용
2. span,div, p태그에 대해 아주 자세히 공부한 티가 납니당 좋아용 ! 다만, 텍스트때문에 p태그로 넣은것같은데 이 부분에서는 한줄단위로 다루면서 여백을 조절하는게 편하니 div로 바꿔보는건 어떨까용?
3. 위 내용과 살짝 연관되는 내용인데 br태그는 사실 잘 쓰지 않습니다..올드한 느낌! inline인 p태그 대신 div3개로 분리해서 자동으로 줄바꿈이 되도록 해보는건 어때용?
5. span, p태그는 인라인요소라 flex에서 다루기 살짝 어렵습니다. div가 flex에서는 좀더 많이 유연하고 잘 이용되니 div를 적극이용해봐용! flex로 반응형디자인할때 유리해집시다!

eunseo9311 committed:
layout framework

eunseo9311 committed:
right box changed

eunseo9311 committed:
style fixed

eunseo9311 committed:
background image input

eunseo9311 committed:
style fixed2

eunseo9311 committed:
style changed

eunseo9311 committed:
layout style changed

eunseo9311 committed:
receipt 1

eunseo9311 committed:
week1 receipt frame

eunseo9311 committed:
week1 end

eunseo9311 committed:
style_week1_html

margin-left: 100px to critic-box

eunseo9311 committed:
style_week1_margin2

margin between critics
top margin for title

eunseo9311 committed:
refecor_week1_html_css

made a css folder to manage style more carefully...
it is not good idea to put style in html

eunseo9311 committed:
chore_week1_css

오른쪽 상단의 별점 박스의 왼쪽 마진을 260px로 띄우려고 노력을 많이 했지만 달라진 것이 없다.. 흑흑

eunseo9311 committed:
feat_week1_css_mediaQuery

css Mobile 버전으로 미디어쿼리를 만들어보고 있다

eunseo9311 committed:
chore_week1_css

미디어쿼리 중앙 정렬 수저함

eunseo9311 committed:
chore_week1_css

별이 들어있는 박스가 여전히 세로형이 되지 않는다!! 도대체 뭐가 문제인지를 모르겠다

eunseo9311 committed:
mediaQuery_exercise.css

eunseo9311 committed:
fixed_week1.css

eunseo9311 committed:
style_week1.css

eunseo9311 committed:
layout_fix 다시 만들기...

eunseo9311 committed:
feat_week1.html

eunseo9311 committed:
new_recipe.html

eunseo9311 committed:
feat_week1.html

eunseo9311 committed:
new_recipet.css_margin

eunseo9311 committed:
new_recipe_nutrition

eunseo9311 committed:
file_name_changed

eunseo9311 committed:
feat_new_recipt_margin

eunseo9311 committed:
feat_new_receipt_number

eunseo9311 committed:
feat_layout_margin

eunseo9311 committed:
feat_recipet_css

eunseo9311 committed:
recipe_css_fix

eunseo9311 committed:
mediaQuery_fix

eunseo9311 committed:
[style] 파일명 변경

주차별로 가독성 있게 파일명을 변경함

eunseo9311 committed:
[chore] week2_todo 구조 잡기

todo 전반적인 초기 구조 설정

eunseo9311 committed:
[fix] week2_todo

띄어쓰기 줄임

eunseo9311 committed:
[fix] html 전반적인 구조 수정

코드의 가독성을 높이기 위해 구조를 수정

eunseo9311 committed:
[fix] 전반적인 css 코드 수정

스타일을 더 자세하게 수정함

eunseo9311 committed:
[chore] todo.js 파일 구조 짜기

전반적인 js 구조를 만들었다

xxoyya reviewed (commented):
수고하셨습니다!!😍

1. width가 738px이 됐을 때 바뀌게 되는데 바뀌었을 때, 화면의 중앙에 위치하지 않아요. body에 스타일을 주어 중앙에 정렬되도록 만들어 보는 건 어떨까용?

2. 반응형 전 기본 페이지에서 리뷰박스의 위치가 너무 내려가 있는데 star-box-1의 스타일 값에서 margin-top이 200px로 되어 있어서 그런 것 같아요. margin-top 값을 조금 줄여도 좋을 것 같습니다!

3. 타이틀에 text-align 으로 center에 올 수 있도록 만드려고 한 것 같은데 적용이 되지 않는 것 같아요. 그 이유가 html 코드를 작성할 때 h1에 style을 바로 text-align: left라고 작성하여 반응형을 했을 때 center가 적용이 되지 않는 것 같습니다. h1에 바로 style을 주지 않고 따로 클래스를 만들어서 left 스타일을 주는 것도 좋을 것 같아용!

eunseo9311 committed:
[form fixed] 전반적인 코드 개선

일차적으로 코드를 전부 개선함, 구조적인 어려움이 많아서 깔끔하게 동작이 되도록 바꾸었다

eunseo9311 committed:
[style] 이미지 이름 변경

구조를 변경하면서 이미지 이름 변경

eunseo9311 committed:
[fix] 체크박스 이벤트 우선순위 변경

체크박스가 표시된 항목들이 체크박스가 표시되지 않은 항목들보다 항상 위쪽에 위치하도록 체크박스 이벤트의 우선순위를 변경하였다

eunseo9311 committed:
[fix] 체크박스 상태 정렬 reFixed

이게 체크박스 상태 정렬을 진짜 바꾼것이다! 아까 코드는 sunIcon이랑 moonIcon을 누르면 dark테마랑 light테마까지 바뀌는 오류가 있었는데 그걸 해결한 것이다

eunseo9311 committed:
[chore] 모든 파일 이름 수정

파일 이름들을 전부 가독성 있게 수정함
너무 이름이 길어지는 경향이 있는거 같아서 가능한 주차별_이름_기능 위주로 수정해봤다

eunseo9311 committed:
[chore] useState 구조잡기

리액트 useState 구조잡기

eunseo9311 committed:
[구조변경] 파일 이름 변경

파일 이름 변경 및 통일

eunseo9311 committed:
[fix] 이미지 형식 수정, 테마 수정

이미지 크기가 체크박스 개수에 비례해서 자꾸 늘어나는 것 같아서 이미지 크기를 고정해주는 방식을 선택함,
테마에 따른 이미지 형식이 적용 안 된 부분을 수정함

eunseo9311 committed:
[코드추가] useEffectExample1

eunseo9311 committed:
[코드추가] useEffectExample2

eunseo9311 committed:
[코드 추가] UseRefExample1

eunseo9311 committed:
[코드 추가] UseRefExample2

eunseo9311 committed:
[코드 추가] UseRefExample3

eunseo9311 committed:
[코드추가] UseRefExample4

eunseo9311 committed:
[코드 추가] UseStateExample1,2

eunseo9311 committed:
[chore] App.js 내용 추가

App.js에 이걸 추가하고 연결해야 돌아가는 것이다

eunseo9311 committed:
[fix] 변수명 변경

useEffectEx1 -> useEffectExample1 이런식으로 바꿈

eunseo9311 committed:
[fix] Timer.js 내용 추가

eunseo9311 committed:
[fix] Timer.js 파일 위치 수정

src 안에 해당 component 파일을 넣어야 한다. 이걸 밖에 넣어서 계속 못 찾고 고생한 것임

eunseo9311 committed:
[fix] background-image 100% 비율 맞추기

이거를 100% 비율로 안하고 14440px 이런걸로 하면 결과적으로 배율을 작게하면 이미지가 깨지게 됩니다, 주의할 것

eunseo9311 committed:
[fix] deleteIcon 연결

이름을 delete 로 해놔서 연결이 안 되었던 거임 이름 바꿈

xxoyya reviewed (commented):
전체적으로 코드를 매우x100 잘 짜신 것 같아요!!

각 todo item에 드래그 했을 때 색이 변하는 기능 추가한 것과 완료된 todo들이 자동적으로 가장 위로 올라가는 기능도 좋은 것 같아용❤️
이번 과제에서는 딱히 수정해야 할 부분이 보이지 않았어요!

수고하셨습니다!!
